core/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -220,6 +227,11 @@ public IndexNameExpressionResolver(Settings settings) {
         return concreteIndices.toArray(new Index[concreteIndices.size()]);
     }
 
+    private static IllegalArgumentException aliasesNotSupportedException(String expression) {
+        return new IllegalArgumentException("The provided expression [" + expression + "] matches an " +
+                "alias, specify the corresponding concrete indices instead.");
```

Comment:
Wouldn't it be more in sync with the majority of ES msgs if `indices` is replaced with `index` in `specify the corresponding concrete indices instead.`